config/config_mgmt.py

```diff
@@ -577,7 +576,7 @@ def _searchKeysInConfig(self, In, Out, skeys):
         '''
         found = False
         if isinstance(In, dict):
-            for key in In.keys():
+            for key in list(In.keys()):
```

Comment:
```suggestion
            for it in In:
```

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.